core/controllers/editor.py

```diff
@@ -76,15 +88,66 @@ class ExplorationHandler(EditorHandler):
     """Page with editor data for a single exploration."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'v': {
+                'schema': {
+                    'type': 'int'
+                },
+                'default_value': None
+            },
+            'apply_draft': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
We shouldn't pass booleans as strings.

Reply:
It is mostly used in number of the places, one of the example is: [here](https://github.com/oppia/oppia/blob/14e4342db42ce9c3ef9ca811a7535619c02a4d0e/core/controllers/reader.py#L796)
Since they are in the form of unicode, I can add choices ['true', 'false'].
Looking forward for your suggestion.

Reply:
In that case, can't you just add `default_value: 'False'`?

Reply:
Updated based on discussion, please take a look.